doc/es_ES/2fa.markdown

```diff
@@ -24,7 +24,7 @@ Instalacion
 2. Click a la izquierda en **Two factor authentication** y selecionar la caja
 3. Una key secret es geradada para ti
 
-![2FA](screenshots/2fa.png)
+![2FA](https://kanboard.net/screenshots/documentation/2fa.png)

```

Comment:
The image links have to be relative, not absolute. Thank you.

Los enlaces de imagen tienen que ser relativos, no absolutos. Gracias.